tests/end2end/features/test_tabs_bdd.py

```diff
@@ -19,3 +19,8 @@
 
 import pytest_bdd as bdd
 bdd.scenarios('tabs.feature')
+
+
+@bdd.when("I wait for a prompt")
+def wait_for_prompt(quteproc):
+    quteproc.wait_for(message='Asking question *')
```

Comment:
You can simply do `And I wait for "Asking question *" in the log` in the `feature` file (though you might want to include some portion of the question text as well)